pkg/monitors/load/load.go

```diff
@@ -0,0 +1,70 @@
+package load
+
+import (
+	"context"
+	"runtime"
+	"time"
+
+	"github.com/shirou/gopsutil/load"
+	"github.com/signalfx/golib/v3/datapoint"
+	"github.com/signalfx/signalfx-agent/pkg/core/config"
+	"github.com/signalfx/signalfx-agent/pkg/monitors"
+	"github.com/signalfx/signalfx-agent/pkg/monitors/types"
+	"github.com/signalfx/signalfx-agent/pkg/utils"
+	"github.com/sirupsen/logrus"
+	log "github.com/sirupsen/logrus"
+)
+
+func init() {
+	if runtime.GOOS != "windows" {
```

Comment:
would it make sense add a log message saying it isn't supported in case this monitor is configured on windows?

Reply:
It will essentially say that the monitor doesn't exist if they try and run it on windows and then if they look in the docs it clearly says that it only works on Linux, so I think that is enough.